core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -209,7 +209,7 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
 
```

Comment:
Do we need to use `inTransactionWithAttrTempTable` for `copyEntities`?

Reply:
I just stepped through the code and it's a bit convoluted but it doesn't look like it's needed. It just does a direct save rather than rewriting anything using the temp tables